examples/powershell/nsx-t_tag-sync-vro/stack.yml

```diff
@@ -0,0 +1,19 @@
+provider:
+  name: openfaas
+  gateway: https://phxlvveba01.itplab.local
+functions:
+  nsxttagsync-vro:
+    lang: powercli
+    handler: ./handler
+    image: cmstraka/veba-powershell-vro-tag-sync-to-nsxt:latest
```

Comment:
Please name this `vmware/veba-powershell-vro-tag-sync-to-nsxt:latest` since we'll build this container and host it upon approval

Reply:
Done!